src/interfaces/csharp/swig_typemaps.i

```diff
@@ -309,7 +307,7 @@ TYPEMAP_STRINGFEATURES(float64_t, double, double)
 }
 
 %typemap(out) shogun::SGStringList<char> {
-	shogun::SGString<char>* str = $1.strings;
+	shogun::SGVector<char>* str = $1.strings;
```

Comment:
I wonder: as an intermediate step (before using std::vector for SGStringList), why don't we rename SGStringList to SGVectorList?

Also, I guess we should rename `strings` to `vectors`